src/components/shared/Entity/Shard/Hydrator.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEntityType } from 'context/EntityContext';
+import useShardsList from 'hooks/useShardsList';
+import { useEffect } from 'react';
+import {
+    useShardDetail_setError,
+    useShardDetail_setShards,
+} from 'stores/ShardDetail/hooks';
+import { BaseComponentProps } from 'types';
+
+interface Props extends BaseComponentProps {
+    lastPubId: string;
+    catalogName: string;
+}
+
+function ShardHydrator({ catalogName, children, lastPubId }: Props) {
```

Comment:
This hydrator was made by pulling the code that used to sit above the component in the panel. Same with the other new hydrators.